examples/Playground.hs

```diff
@@ -1,4 +1,6 @@
-module Main where
+{-# OPTIONS -Wno-unused-top-binds #-}
```

Comment:
Is there's something unused? I guess this module shouldn't contain unused definitions and everything in this module should be used unless it's redundant.

Reply:
Yeah it's pretty weird, I think it's an actual GHC bug. 
It's because of `newtype N = N Text`, the compiler says the `N` _constructor_ is never used, even though the `N` _type_ is used once in `data Test = Test { ..., testN  :: N ... }`. I couldn't find a cleaner way to go around this.

Reply:
@jiegillet You can submit bug to GHC with minimal reproducible example :slightly_smiling_face:
Feel free to ask any questions about your example or bug-reporting process!

Reply:
I've never done that before, exciting, I'll try.
Seriously, I've learned so much and found out so many things in the last 6 weeks, I cannot thank you enough!

Reply:
Seems there's already a ticket [here](https://ghc.haskell.org/trac/ghc/ticket/10347). It doesn't look like it's considered high priority though...

Reply:
@jiegillet You can leave the comment under that issue as well. Usually, the more comments under bug the more chances it will be closed :slightly_smiling_face:

Reply:
Done! 
In the mean time, do I leave the `{-# OPTIONS -Wno-unused-top-binds #-}`?

Reply:
@jiegillet Yes, let's keep this option.